client/src/components/Profile/index.tsx

```diff
@@ -36,9 +38,10 @@ export interface IProfileProps {
 
 export default class Profile extends React.Component<IProfileProps> {
 
-    public state: { loaded: boolean, modal: boolean } = {
+    public state: { loaded: boolean, modal: boolean, modalOpen: boolean } = {
         loaded: false,
         modal: false,
+        modalOpen: false,
```

Comment:
Modal, modalOpen

Pls give more clear names for both params

For example, followersModal

Reply:
fixed